src/Dockerfile

```diff
@@ -22,6 +22,8 @@ RUN rm -f requirements.txt
 STOPSIGNAL SIGINT
 EXPOSE 8080
 
+ENV S3GW_UI_LOCATION "/s3store"
```

Comment:
Wouldn't it be better to default it to `/`?

Reply:
Good catch, this is a leftover from mocking this up early on. "/" as default makes more sense because then the chart doesn't need to be changed as well.

I'll also be changing `location` to `path` as per above suggestion. I think `location` is what it's called in the Nginx config files.